package-lock.json

```diff
@@ -672,7 +765,20 @@
       }
     },
     "fabric8-hypothesis-core-image": {
-      "version": "github:fabric8-hdd/fabric8-hypothesis-core-image#07f58d86f1df31f0f440f2f6a2dfe8950af49d5f"
+      "version": "github:fabric8-hypothesis/fabric8-hypothesis-core-image#592bd3aea837733cf36fe93c4706f242590dedcd"
+    },
+    "fabric8-hypothesis-measurements-api": {
+      "version": "github:fabric8-hypothesis/fabric8-hypothesis-measurements-api#e468622aab45c16e093ca3b7759ad9177e478caf",
+      "requires": {
+        "express": "4.16.3",
+        "fabric8-hypothesis-core": "github:fabric8-hdd/fabric8-hypothesis-core#88e5774c17c15491366e322bc698586f4e74eb7e",
```

Comment:
please delete this file and regenerate this file by running `npm install`

Seems like 2 places the replacement is not done properly